File: src/main/java/me/santio/uhc/models/ScenarioData.java
package me.santio.uhc.models;

import lombok.Getter;
import lombok.Setter;
import org.apache.commons.lang.WordUtils;

@SuppressWarnings("unused")
public class ScenarioData {
    @Setter @Getter private String description;
    @Setter @Getter private Object value = null;
    @Getter private final String key;
    
    public ScenarioData(String key, String description) {
        this.key = key;
        this.description = description;
    }
    
    public ScenarioData(String key, String description, Object value) {
        this.key = key;
        this.description = description;
        this.value = value;
    }
    
    public String[] getWrappedDescription() {
        return WordUtils.wrap(getDescription(), 30).split("\n");
    }
}

File: src/main/kotlin/me/santio/uhc/models/ScenarioData.kt
package me.santio.uhc.models

import org.apache.commons.lang.WordUtils

class ScenarioData(val key: String, val description: String, var value: Any?) {

    val wrappedDescription: Array<String>
        get() = WordUtils.wrap(description, 30).split("\n").toTypedArray()
}